packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -103,12 +104,30 @@ class DDSMastheadL1 extends StableSelectorMixin(LitElement) {
   @property()
   l1Data?: MastheadL1;
 
+  /**
+   * The URL path of the current document root.
+   */
+  @property()
+  currentUrlPath?: string = root.location?.href;
```

Comment:
Thinking out loud. Would `root.location.href` ever change after this is initialized in a way we would have to be concerned about? Thinking of, for example, a single page app that manages it's own routing client side for instance.

Reply:
Good point, makes sense. Thannks John.